### PR TITLE
Update: UI changes for transcript cards

### PR DESCRIPTION
**Pull request by priyajeet**
Also adds error cards
Style changes for preview sidebar
Package updates

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: src/components/ContentPreview/ContentPreview.scss [hunk 1]
```diff
@@ -3,13 +3,29 @@
...
+        flex: 1;
+    }
+
+    .bcpr-content {
+        flex: 1;
+        // order: 1;
+    }
+
+    .bcs {
+        // border-right: 1px solid $off-white;
```
Comment (wenboyu2) [inline src/components/ContentPreview/ContentPreview.scss]: remove comments?

File: src/components/ContentSidebar/SidebarSection.scss [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 }
 
+.bcs-section {
+    margin: 0 20px 20px 30px;
+    // background-color: $white;
+    // border: 1px solid $off-white;
+    // border-radius: 5px;
+    // box-shadow: 0 0 10px 0 $lightest-black;
+    // margin: 0 10px 15px 15px;
+    // padding: 10px;
```
Comment (wenboyu2) [inline src/components/ContentSidebar/SidebarSection.scss]: remove?

File: src/components/Transcript/Transcript.js [hunk 3]
```diff
@@ -4,95 +4,83 @@
...
+    props: Props;
+    state: State;
+
+    /**
+     * [constructor]
+     *
+     * @private
+     * @return {Transcript}
+     */
+    constructor(props: Props) {
```
Comment (wenboyu2) [inline src/components/Transcript/Transcript.js]: replace this constructor with
`state = { isModalOpen: false };`

File: src/util/download.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+/**
+ * @flow
+ * @file Utility to download string as a file
+ * @author Box
+ */
+
+function download(string: string, name: string) {
```
Comment (wenboyu2) [inline src/util/download.js]: add js doc for these?

File: src/components/ContentSidebar/SidebarSection.scss (hunk 2 above)
Comment (priyajeet) [reply src/components/ContentSidebar/SidebarSection.scss]: Didn't remove these as they are the new design, so uncommenting these will get the new styles. Didnt want to forget what I did before.

Comment (wenboyu2) [reply src/components/ContentSidebar/SidebarSection.scss]: got it, maybe add some comments to explain why we want to keep them? someone else might accidentally remove these

Comment (priyajeet) [reply src/components/ContentSidebar/SidebarSection.scss]: Actually I am just gonna nuke it for now, they are pretty easy to replicate

**Pull request merged**